### ISSUE TITLE
I cannot verify the data for shift_2_days Vs shift_1_days in Influxdb

### ISSUE THREAD

**Issue by ashishkaransingh** [maxsivanov/influxdb-timeshift-proxy#17]
Hi,

Cannot find Data values i see in Grafana in Influxdb.

I see following values in Grafana: 
Max CPU Time shift_2_days: 57
Max CPU Time shift_1_days: 29
Below snip for Grafana shows Max Percent_Processor_Time for shift_2_days Vs shift_1_days.

![image](https://user-images.githubusercontent.com/29729110/46604487-0f395480-cb14-11e8-9710-0fb363b6bac0.png)

Querying in Influxdb:

SELECT max("Percent_Processor_Time") FROM "cpu" WHERE time > now() - 1d  group by time(2h)

![image](https://user-images.githubusercontent.com/29729110/46604606-6b9c7400-cb14-11e8-8b1b-849ffea76384.png)

SELECT max("Percent_Processor_Time") FROM "cpu" WHERE time > now() - 2d  group by time(2h)

![image](https://user-images.githubusercontent.com/29729110/46604615-76570900-cb14-11e8-8f15-d6349fe04fd4.png)

It must be something simple i am missing.

Thanks

**Comment by Codelica** [comment]
Hard to tell from the small Grafana image, but it looks like the time intervals being graphed aren't on whole hour increments like your manual query.  Like that point selected looks like it's at 1:30am (?), perhaps that's the issue.

**Comment by ashishkaransingh** [comment]
> Hard to tell from the small Grafana image, but it looks like the time intervals being graphed aren't on whole hour increments like your manual query. Like that point selected looks like it's at 1:30am (?), perhaps that's the issue.

Hi,
Thank you for you response.
Manual query and query in grafana are similar.
Yes the datapoint is 1:30am, which is correct because i grouped data-points by 2 hours, so it is 11:30 then next 1:30 etc.
I see data-points returned by the queries, but i do not see those values in Influxdb table for me to verify whether the data-points are correct or not.

**Comment by Codelica** [comment]
Well, what I'm saying is that your graph seems to be taking the max over different time periods than your query.  The graph has max info for periods like 11:30-1:30, 1:30-3:30, etc.  Where you raw query for comparison has periods like 10:00-12:00, 12:00-2:00, etc.  So you can't really compare the data as they aren't pulling the max values over the same periods.

**Comment by ashishkaransingh** [comment]
Yeah, i do not know why it is taking the max over different time periods, when essentially the queries are same.
